test/blocks/chart/chart.test.js

```diff
@@ -704,4 +705,36 @@ describe('chart', () => {
   it('getOversizedNumberSize returns miniumum size for more than 6 characters', () => {
     expect(getOversizedNumberSize(100)).to.eql([90, 55, 65]);
   });
+
+  it('Horizontal is the default label orientation', () => {
+    document.body.innerHTML = '<div class="chart line"></div>';
+    const styles = document.querySelector('.chart').classList;
+    expect(getLabelDegree(styles, true)).to.equal(0);
+  });
+
+  it('Sets degree for diagonal labels', () => {
+    document.body.innerHTML = '<div class="chart line mobile-diagonal-labels"></div>';
+    const styles = document.querySelector('.chart').classList;
+    expect(getLabelDegree(styles, false)).to.be.above(0);
+  });
+
+  it('default grid bottom', () => {
+    const options = getChartOptions({});
+    expect(options.grid.bottom).to.equal(90);
+  });
+
+  it('grid bottom for large chart with default labels', () => {
+    const options = getChartOptions({ size: 'large' });
+    expect(options.grid.bottom).to.equal(60);
+  });
+
+  it('grid bottom for large chart with diagonal labels', () => {
+    const options = getChartOptions({ size: 'large', labelDeg: 60 });
+    expect(options.grid.bottom).to.equal(30);
+  });
+
+  it('grid bottom for non-large chart with diagonal labels', () => {
+    const options = getChartOptions({ size: 'small', labelDeg: 60 });
+    expect(options.grid.bottom).to.equal(40);
+  });
```

Comment:
nit, but with the `it` keyword, it seems this testing suite is asking us to use verbs in our test declarations after describes, which you do on line 715 `Sets degree for...`. Maybe something we want to standardize / discuss with milo-community.